core/api/src/config/index.ts

```diff
@@ -13,6 +13,7 @@ export * from "./schema"
 import { ConfigError } from "./error"
 
 import { toDays } from "@/domain/primitives"
+import { QuizzesValue } from "@/app/quiz/config"
```

Comment:
:thinking: shouldn't thits be part of config and not from app layer? it is creating a circular dependency

Reply:
not sure, but I've also been thinking about that. I think this is the config of the quizzes, so we may argue this could be fine? I also try to make the quiz module for self contain.